source/code/source/playfab/PlayFabApiSettings.cpp

```diff
@@ -5,39 +5,33 @@
 
 namespace PlayFab
 {
+    const std::map<std::string, std::string> PlayFabApiSettings::requestGetParams = {
+        { "sdk", PlayFabSettings::versionString }
+    };
+
     PlayFabApiSettings::PlayFabApiSettings() :
-#ifndef DISABLE_PLAYFABCLIENT_API
-        advertisingIdType(PlayFabSettings::advertisingIdType),
-        advertisingIdValue(PlayFabSettings::advertisingIdValue),
-        disableAdvertising(PlayFabSettings::disableAdvertising),
-#endif
-        verticalName(PlayFabSettings::verticalName),
         baseServiceHost(PlayFabSettings::productionEnvironmentURL),
-        titleId(PlayFabSettings::titleId)
+        titleId()
     {
+        // Don't let PlayFabSettings::staticSettings pull titleId from itself
+        if (PlayFabSettings::staticSettings != nullptr)
+        {
+            titleId = PlayFabSettings::staticSettings->titleId;
+        }
     }
 
-    std::string PlayFabApiSettings::GetUrl(const std::string& urlPath, const std::map<std::string, std::string>& getParams) const
+    std::string PlayFabApiSettings::GetUrl(const std::string& urlPath) const
     {
         std::string fullUrl;
         fullUrl.reserve(1000);
```

Comment:
side note: With the removal of getParams here in favor of the object created above, should we consider a finder reserve number for the fullUrl instead of an arbitrary 1000?

Reply:
I'm not opposed to a new feature task to improve that 1000.
But, I am opposed to doing it in this checkin.